data/account-providers/tandem-money-limited.json

```diff
@@ -65,7 +65,7 @@
   ],
   "apiStandards": [],
   "webApplication": false,
-  "mobileApps": [],
+  "mobileApps":true,
```

Comment:
You can't use a boolean for this data point. Follow this format https://github.com/not-a-bank/open-banking-tracker-data/blob/21242c234a16048a37b4807e3c6c2099c52967c4/data/account-providers/revolut.json#L116